app.py

```diff
@@ -61,18 +64,17 @@ def inference(question):
 
         return predicted_text
 
-    if request.method == 'POST':
-        message = request.form['message']
+    message = request.form['message']
```

Comment:
I got rid of this `if` statement because the route is set up to only handle the `POST` method, so it’s impossible for it to be false! That’s all that has functionally changed below here.